<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_30">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>440</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>440</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>440</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>320</width>
      <height>440</height>
     </size>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tabMain">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>439</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>439</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>320</width>
       <height>439</height>
      </size>
     </property>
     <attribute name="title">
      <string>Dzień</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>318</width>
        <height>416</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab7">
     <attribute name="title">
      <string>Tydzień</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_7">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>318</width>
        <height>416</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Miesiąc</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_30">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>318</width>
        <height>416</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>318</width>
        <height>416</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>318</width>
        <height>416</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="nowLabel">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>440</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>320</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpt">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Opcje</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuOpt"/>
  </widget>
  <action name="actionRefresh">
   <property name="text">
    <string>Odśwież</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Ustawienia</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Zakończ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Pomoc</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>resizeItem(bool)</slot>
  <slot>mainlog(QString)</slot>
 </slots>
</ui>
